agave_app/agaveGui.cpp

```diff
@@ -470,7 +469,7 @@ agaveGui::open(const std::string& file, const ViewerState* vs)
   int sceneToLoad = vs ? vs->m_currentScene : 0;
   int timeToLoad = vs ? vs->m_currentTime : 0;
 
-  if (file.find("http") == 0) {
+  if (file.find("http") == 0 || file.find("zarr") != std::string::npos) {

```

Comment:
is the http case needed?  Could you just check or files ending in zarr?  Or do we have S3 objects that are zarrs named without the extension?

Reply:
It's certainly true that the only thing agave can load over http is zarr, so if it's http we have to try the zarr loader. 
 
For local data, I'm just making a basic vague guess that if zarr is in the path, it should be one. Sometimes you can point to a subdirectory of a zarr store and still read a zarr.   I'm totally open to refining this later when I revisit data loading to make it all more streamlined from the front end.